system/ee/ExpressionEngine/Addons/fluid_field/views/field.php

```diff
@@ -14,14 +17,20 @@
     			<a href class="dropdown__link dropdown__link--danger js-fluid-remove"><i class="fas fa-fw fa-trash-alt"></i> <?=lang('delete')?></a>
     		</div>
     		<?php else: ?>
-    			<button type="button" class="fluid__item-tool js-fluid-remove danger-link" title="<?=lang('remove')?>"><i class="fas fa-fw fa-trash-alt"></i></button>
+    			<button type="button" class="fluid__item-tool js-fluid-remove danger-link" title="<?=lang('remove')?>">
+    				<span class="sr-only">remove icon</span>
```

Comment:
should be better to use language file here. Maybe this could be just `<?=lang('remove')?>` - same as title for it?